tests/dmn/test_parser.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path
+
+from django_camunda.dmn.parser import Parser
+
+TEST_DRD_XML = Path(Path(__file__).parent / "data" / "drd.xml").read_bytes()
```

Comment:
```suggestion
TEST_DRD_XML = (Path(__file__).parent / "data" / "drd.xml").read_bytes()
```
should be sufficient :) the `/` operator returns a new path instance already